exercises/practice/markdown/.meta/config.json

```diff
@@ -1,6 +1,35 @@
 {
   "blurb": "Refactor a Markdown parser",
   "authors": [],
```

Comment:
Another empty one, added in https://github.com/exercism/elixir/commit/b4dac08e9dde0956bac0a994827e2c96a89b7451 by Yeong Sheng, Tan